src/components/ContentSidebar/ActivityFeed/approval-comment-form/ApprovalCommentForm.js

```diff
@@ -0,0 +1,261 @@
+/**
+ * @flow
+ * @file Component for Approval comment form
+ */
+
+import React, { Component } from 'react';
+import type { Node } from 'react';
+import cx from 'classnames';
+import { EditorState } from 'draft-js';
+import { FormattedMessage, injectIntl, intlShape } from 'react-intl';
+
+import Avatar from 'box-react-ui/lib/components/avatar/Avatar';
+import Form from 'box-react-ui/lib/components/form-elements/form/Form';
+import DraftJSMentionSelector, {
+    DraftMentionDecorator
+} from 'box-react-ui/lib/components/form-elements/draft-js-mention-selector';
+import commonMessages from 'box-react-ui/lib/common/messages';
+
+import AddApproval from './AddApproval';
+import CommentInputControls from './CommentInputControls';
+import messages from '../../../messages';
+import type { SelectorItems, User } from '../../../../flowTypes';
+
+import './ApprovalCommentForm.scss';
+
+type Props = {
+    approverSelectorContacts: SelectorItems,
+    className: string,
+    createComment: Function,
+    createTask: Function,
+    updateTask: Function,
+    getApproverContactsWithQuery: Function,
+    getMentionContactsWithQuery: Function,
+    intl: intlShape.isRequired,
```

Comment:
^ thats not a flowtype, its the old proptype
I think use any, or see the flow-typed for react-intl

Reply:
done. Using any.